week-1/the-pokes/tictactoe.rb

```diff
@@ -0,0 +1,119 @@
+puts "---- Bienvenidos a Tic Tac Toe Codeable ----"
+puts "Reglas del juego:"
+puts "--------------------------------------------"
+puts "1. Los jugadores eligiran la posicion donde
+  marcaran"
+puts "--- Tablero ---"
+
+puts "\n 1 | 2 | 3 "
+puts "-----------"
+puts " 4 | 5 | 6 "
+puts "-----------"
+puts " 7 | 8 | 9 "
+
+puts "\nEmpecemos!!!"
+
+puts "\n   |   |   "
+puts "-----------"
+puts "   |   |   "
+puts "-----------"
+puts "   |   |   "
+
+COMBINATIONS_WIN = [
+  [1,2,3],  
+  [4,5,6],
+  [7,8,9],
+  [1,4,7],
+  [2,5,8],
+  [3,6,9],
+  [1,5,9],
+  [3,5,7]
+]
+
+TABLE = [
+  1,
+  2,
+  3,
+  4,
+  5,
+  6,
+  7,
+  8,
+  9
+]
+
+def notPlayed(value)
+  arrNP = TABLE - value
+  return arrNP 
+end
+
+def turnHuman(notplayed)
+    value = 0
+    loop do
+      puts "\nEs el turno del Humano"
+      value = gets.chomp.to_i
+      break if ( ( notplayed.include?(value)) && (value >= 1) && (value <=9) )
+    end
+    return value
+end
+
+def turnBot(value)
+  if(value != [])
+    value.sample
+  end
+end
+
+def tictactoe()
+  
+  value_human = []
+  value_bot = []
+
+  band = false
+
+  player = "X"
+  bot = "O"
+
+  table = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
puedes poner table = [" "] para crear un array vacio

Reply:
Maybe we can use this
`table = Array.new(8," ")`
-> This is used for print the table tic - tac - toe